mix.exs

```diff
@@ -34,7 +34,8 @@ defmodule AtEx.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:tesla, "~> 1.2.1"},
+      {:tesla, "~> 1.4.3"},
+      {:hackney, "~> 1.17"},
```

Comment:
@MuriukiPM since `hackney` is not used internally on `at_ex`, it will be safe to remove. This will in turn make the ci pipeline pass. cc @manuelgeek  
```suggestion
```